packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -156,6 +157,7 @@ export const CallWithChatControlBar = (props: CallWithChatControlBarProps & Cont
       <Stack.Item grow>
         <CallAdapterProvider adapter={props.callAdapter}>
           <Stack horizontalAlign="center">
+            {HiddenFocusStartPoint()}
```

Comment:
Could just use this as a component instead of a function like <HiddenFocusStartPoint />, which react might have some optimizations on its own